src/js/jsx/shared/Datalist.jsx

```diff
@@ -90,7 +90,7 @@ define(function (require, exports, module) {
                 defaultSelected: null,
                 live: true,
                 startFocused: false,
-                placeholderText: "",
+                placeholderText: "[missing font]",
```

Comment:
Also, this is in a shared component, so it needs a new prop to generalize it otherwise all datalists will have "missing font" placeholder text.